src/build/fish_completions.zig

```diff
@@ -114,7 +113,6 @@ fn writeFishCompletions(writer: anytype) !void {
             } else try writer.writeAll(" -f");
 
             switch (@typeInfo(opt.type)) {
```

Comment:
```suggestion
            switch (@typeInfo(opt.type)) {
               .Bool => {},
```

Explicitly do nothing rather than forget boolean values need no arguements?

Reply:
Agreed, explicit handling is better.